lib/process-services-cloud/src/lib/start-task-cloud/components/people-cloud/people-cloud.component.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { UserCloudModel } from '../../models/user-cloud.model';
+
+describe('PeopleCloudComponent', () => {
+    let component: PeopleCloudComponent;
+    let fixture: ComponentFixture<PeopleCloudComponent>;
+    let element: HTMLElement;
+    let service: StartTaskCloudService;
+    let getRolesByUserIdSpy: jasmine.Spy;
+    let getUserSpy: jasmine.Spy;
+
+    setupTestBed({
+        imports: [ProcessServiceCloudTestingModule, StartTaskCloudTestingModule],
+        providers: [StartTaskCloudService, LogService]
+    });
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(PeopleCloudComponent);
+        component = fixture.componentInstance;
+        element = fixture.nativeElement;
+        service = TestBed.get(StartTaskCloudService);
+        getRolesByUserIdSpy = spyOn(service, 'getRolesByUserId').and.returnValue(of(mockRoles));
+        getUserSpy = spyOn(service, 'getUsers').and.returnValue(of(fakeUsers));
+        fixture.detectChanges();
+    });
+
+    it('should create PeopleCloudComponent', () => {
+        expect(component instanceof PeopleCloudComponent).toBeTruthy();
+    });
+
+    it('should able to fetch users', () => {
+        fixture.detectChanges();
+        expect(getUserSpy).toHaveBeenCalled();
+    });
+
+    it('should able to fetch roles by user id', () => {
+        fixture.detectChanges();
+        expect(getRolesByUserIdSpy).toHaveBeenCalled();
+    });
+
+    it('should return empty display name for missing model', () => {
+        expect(component.getDisplayName(null)).toBe('');
+    });
+
+    it('should return full name for a given model', () => {
+        let model = <UserCloudModel> { firstName: 'John', lastName: 'Doe'};
+        expect(component.getDisplayName(model)).toBe('John Doe');
+    });
+
+    it('should skip first name for display name', () => {
+        let model = <UserCloudModel> { firstName: null, lastName: 'Doe'};
+        expect(component.getDisplayName(model)).toBe('Doe');
+    });
+
+    it('should skip last name for display name', () => {
+        let model = <UserCloudModel> { firstName: 'John', lastName: null};
+        expect(component.getDisplayName(model)).toBe('John');
+    });
+
+    it('should show the users if the typed result match', async(() => {
+        component.users$ = of(fakeUsers);
+        fixture.detectChanges();
+        let inputHTMLElement: HTMLInputElement = <HTMLInputElement> element.querySelector('input');
+        inputHTMLElement.focus();
+        inputHTMLElement.dispatchEvent(new Event('input'));
+        inputHTMLElement.dispatchEvent(new Event('keyup'));
+        inputHTMLElement.dispatchEvent(new Event('keydown'));
+        inputHTMLElement.value = 'M';
+        fixture.detectChanges();
+        fixture.whenStable().then(() => {
+            fixture.detectChanges();
+            expect(fixture.debugElement.query(By.css('mat-option'))).toBeDefined();
```

Comment:
instead of checking that the option exist can we check that the user is mock-username1